### PR TITLE
Introduce get_scheme and get_hostname on Url classes

### PR DESCRIPTION
**Pull request by rixx**
Closes maybe, but at least references #4.

I hope this provides a fairly readable change, but please request changes if anything seems wonky to you!

### CODE REVIEW

File: urlman.py [hunk 1]
```diff
@@ -31,10 +31,12 @@ class UrlsMetaclass(type):
     """
     def __new__(self, name, bases, attrs):
         # Collect patterns off of the attrs
+        callables = [
```
Comment (andrewgodwin) [inline urlman.py]: I'd probably move this into a class variable rather than initialising it every time.

**Review by andrewgodwin — COMMENTED** [review]
One small change, otherwise looks good!

File: urlman.py (hunk 1 above)
Comment (rixx) [reply urlman.py]: That makes more sense, definitely. Pushed :)

**Comment by andrewgodwin** [comment]
Ta!

**Pull request merged**